### DISCUSSION TITLE
How to move data from existing cluster to postgresql-operator

### DISCUSSION THREAD

**Discussion by mesiu84** [zalando/postgres-operator#2328] [Q&A]
Is there an instruction somewhere about how to migrate to postgresql-operator?
The only one I've found until now is this one:
https://blog.palark.com/our-experience-with-postgres-operator-for-kubernetes-by-zalando/ (didn't try it yet, I'm still in development mode)
Anyone else trying to migrate DB using some other procedure?

**Comment by eebrandonmorris** [comment]
Any update on if that procedure worked or not?
